src/colors.rs

```diff
@@ -155,9 +155,22 @@ impl ColoredLevelConfig {
     ///
     /// [`ColoredLevelConfig::default`]: #method.default
     #[inline]
+    #[cfg(not(windows))]
     pub fn new() -> Self {
         Self::default()
     }
+    
+    /// Creates a new ColoredLevelConfig with the default colors.
+    ///
+    /// This matches the behavior of [`ColoredLevelConfig::default`].
+    ///
+    /// [`ColoredLevelConfig::default`]: #method.default
```

Comment:
To avoid duplicating the signature and documentation, what would you think of adding the `#[cfg]` on the `let _ = colored::control::...` statement instead?

We could use:

```
#[cfg(windows)]
{
    let _ = colored::control::...;
}
``

and that should just enable/disable that line of code rather than having to duplicate the method.

Reply:
Sounds like a more appropriate solution will change 👌